### PR TITLE
Add a getting started section

### PR DESCRIPTION
**Pull request by saschatimme**
Added a getting started section.
Feedback appreciated!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,110 @@
...
 
+## Getting started
+
+Great that you want to use Reason with React Native! To get everything running are just a couple of steps.
+Let's assume that you already have a React Native project. Otherwise follow the React Native [instructions](http://facebook.github.io/react-native/docs/getting-started.html) until you have your app running.
+
+1. Install [Bucklescript](https://github.com/bloomberg/bucklescript) (the Reason -> JS compiler), [Reason-React](https://github.com/reasonml/reason-react) and `bs-react-native`:
+```sh
+# substitute yarn with npm if you prefer
+yarn add bs-platform reason-react https://github.com/BuckleTypes/bs-react-native.git
```
Comment (wokalski) [inline README.md]: `bs-platform` is a dev dependency

File: README.md [hunk 2]
```diff
@@ -1,5 +1,110 @@
...
+```json
+"scripts": {
+  ...
+  "build": "bsb -make-world -clean-world",
+  "watch": "bsb -make-world -clean-world -w"
+}
+```
+Now you can build all your (so far nonexsisting) Reason code with `yarn run build` or enter the watch mode with `yarn run watch`.
+
+5. Start the watcher with `yarn run watch`.
```
Comment (wokalski) [inline README.md]: Repeats the information from above. What about this:

5. Now you can build all your (so far nonexsisting) Reason in two modes:
  - `yarn run build` performs a single build
  - `yarn run watch` enters the watch mode

File: README.md [hunk 3]
```diff
@@ -1,5 +1,110 @@
...
+6. Now we come to the fun stuff! We will rebuild the default app component with Reason. Create a new file `re/app.re` and make it look like this:
+```reason
+open ReactNative;
```
Comment (wokalski) [inline README.md]: Maybe let's shorten this to a simple `Hello World`? I can see that you made it the same as `react native` intro but I think it's a lot of things a beginner might not know.

```reason
open ReactNative;
let helloWorld () => 
  <View style=Style.(style [flex 1., justifyContent `center, alignItems `center])>
    <Text value="Hello World" />
  </View>;
```

```js
 import { helloWorld } from "./lib/js/re/app.js";
 import React from "react";
 import {
   AppRegistry
 } from 'react-native';
 
 AppRegistry.registerComponent('MyAwesomeProject', helloWorld);
```

File: README.md (hunk 2 above)
Comment (saschatimme) [inline README.md]: Changed it to your version. And the introduced again repetition! :D

Comment (saschatimme) [inline README.md]: Just want to avoid, that someone forgets to build the stuff and gets a big red error screen

File: README.md (hunk 3 above)
Comment (saschatimme) [inline README.md]: You are probably right

Comment (saschatimme) [inline README.md]: Gave it a second spin!

**Comment by saschatimme** [comment]
Gave it a second spin!
